### PR TITLE
Update PRC modeling.

### PR DESCRIPTION
**Pull request by davidlehn**
- Add marriage related properties to context.
- Add `permanentResidentCard` property.
- Add `PermanentResidentCardCredential` type.
- Use `PermanentResidentCardCredential` for a top level VC type.
- Main credential subject is Person,PermanentResident.
- Subject has a `permanentResidentCard`.
- Move PRC properties to PRC object.
- Update example and docs.

### CODE REVIEW

**Comment by davidlehn** [comment]
Feedback needed:
- Is this modeling better?
- Should `image` be for the person or the card?
- Should `name` and `description` be on the card only?  What are good example values?
- Do person values need to be duplicated in the card?
- Do locations need finer detail or is general locality enough?

File: index.html [hunk 1]
```diff
@@ -199,7 +196,6 @@ <h3>Example</h3>
     "additionalName": "JACOB",
     "familyName": "SMITH",
     "gender": "Male",
-    "image": "data:image/png;base64,iVBORw0KGgo...kJggg==",
     "residentSince": "2015-01-01",
     "lprCategory": "C09",
     "lprNumber": "999-999-999",
```
Comment (dlongley) [inline index.html]: These should be on the card instead of the person, I'd think.

File: index.html [hunk 2]
```diff
@@ -230,6 +221,19 @@ <h3>Example</h3>
       "addressLocality": "Nassau",
       "addressRegion": "New Providence",
       "addressCountry": "BS"
+    },
+    "permanentResidentCard": {
+      "type": "PermanentResidentCard",
+      "identifier": "83627465",
+      "name": "Permanent Resident Card",
+      "description": "Government of Example Permanent Resident Card.",
```
Comment (dlongley) [inline index.html]: I think these should be on the top-level credential object. They are needed for general rendering.

Comment (davidlehn) [reply index.html]: Moved.  Defined them here for PRCCred though in VCDM v2 they are defined by default.  They could be added on the PRC itself as well if needed.

File: index.html [hunk 3]
```diff
@@ -185,11 +185,10 @@ <h3>Example</h3>
     <span class="comment">// e.g., https://uscis.gov/prc/v1</span>
   ],
   "id": "https://issuer.oidp.uscis.gov/credentials/83627465",
-  "type": ["VerifiableCredential", "PermanentResidentCard"],
+  "type": ["VerifiableCredential", "PermanentResidentCardCredential"],
+  "name": "Permanent Resident Card Credentials",
```
Comment (msporny) [inline index.html]:
```suggestion
  "name": "Permanent Resident Card",
```

File: index.html [hunk 4]
```diff
@@ -185,11 +185,10 @@ <h3>Example</h3>
...
+  "description": "Government of Example Permanent Resident Card Credentials.",
```
Comment (msporny) [inline index.html]:
```suggestion
  "description": "Permanent Resident Card for the Government of Example.",
```

**Review by msporny — APPROVED** [review]

**Pull request merged**